plugin.cpp

```diff
@@ -204,8 +211,19 @@ PLUGIN_HANDLE plugin_init(ConfigCategory* config,
 					return NULL;
 				}
 			}
+			else if (actionStr == "remove")
+			{
+				actn = action::REMOVE;
+
+                                if (iter->HasMember("datapoint"))
+                                        datapoint = (*iter)["datapoint"].GetString();
```

Comment:
To be really safe testing if (*iter)["datapoint"].IsString() would be good to add.

Reply:
fixed.